tests/all/test_instruction_values.rs

```diff
@@ -88,6 +92,7 @@ fn test_operands() {
     let free_call_param = free_instruction.get_operand(0).unwrap().left().unwrap();
 
     assert_eq!(bitcast_use_value, free_call_param);
+    */
```

Comment:
Why was this commented out?

Reply:
Perhaps, I was too hasty in removal of the bitcase mentions.
Let's try the CI with this uncommented, and I'll wrap it into a conditional if it fails.